game.py

```diff
@@ -0,0 +1,46 @@
+import pieces
+import numpy as np
+
+class Game():
+    def __init__(self):
+        self.board = [[None for i in range(8)] for j in range(8)]
+        self.initialize_white()
+        self.initialize_black()
+    
+    def initialize_white(self):
+        #Initialize pawns
+        for j in range(8):
+            self.board[6][j] = pieces.Pawn('b', 6, j)
+        #Rooks
+        self.board[7][0] = pieces.Rook('b', 7, 0)
+        self.board[7][7] = pieces.Rook('b', 7, 7)
+        #Knights
+        self.board[7][1] = pieces.Knight('b', 7, 1)
+        self.board[7][6] = pieces.Knight('b', 7, 6)
+        #Bishops
+        self.board[7][2] = pieces.Bishop('b', 7, 2)
+        self.board[7][5] = pieces.Bishop('b', 7, 5)
+        #Royalty
+        self.board[7][3] = pieces.Queen('b', 7, 3)
+        self.board[7][4] = pieces.King('b', 7, 4)
```

Comment:
Both of these functions are very similar. I would think that a more modularized way of doing this is:
```
```suggestion
    from enum import Enum
    class Color(Enum):
         WHITE = "White"
         BLACK = "Black"
    
    def initialize_player_pieces(self, color: COLOR):
        if color == COLOR.WHITE:
            i = 7
        else:
            i = 0
        #Initialize pawns
        for j in range(8):
            self.board[6][j] = pieces.Pawn(color, 6 if color == color.WHITE else 1, j)
        #Rooks
        self.board[i][0] = pieces.Rook(color, i, 0)
        self.board[i][7] = pieces.Rook(color, i, 7)
        ...
```

This + keeping in mind that the above comment about how to access pieces based on like A4 instead of by two indices